mix.exs

```diff
@@ -8,7 +8,7 @@ defmodule Plaid.Mixfile do
   def project do
     [
       app: :plaid,
-      version: "1.7.2",
+      version: "2.4.0",
```

Comment:
I'm not sure if I need to be doing this, this was copied over from the original PR on the main repo. 

But, I think this version bump doesn't quite make sense for us since we haven't been bumping it the whole time. Let me know if you think I should remove this or it doesn't really matter since we point out dependency on `brex-head` anyways instead of a specific version.

Reply:
Yeah this version bump isn't necessary - the way that our code in `credit_card` knows what code to pull for this dependency is the git branch as you said, and the git SHA in the `mix.lock` file

I'd recommend just leaving the version as-is (1.7.2)